server/bleep/Cargo.toml

```diff
@@ -89,6 +87,7 @@ replace_with = "0.1.7"
 relative-path = "1.7.2"
 git2 = "0.15.0"
 dunce = "1.0.3"
+histogram = { version = "0.6.9", optional = true }
```

Comment:
Can you add a comment explaining that this is used for logging when the debug feature is enabled?

Reply:
`optional` dependencies are usually picked up by `features`, like here: https://github.com/BloopAI/bloop/pull/9/files#diff-f8919bb455c366daf0d703faef73f63e745e0a06dde47f6e62b5eb88c78687b1R10